src/model/Series.ts

```diff
@@ -575,8 +575,8 @@ class SeriesModel<Opt extends SeriesOption = SeriesOption> extends ComponentMode
         if (data.hasItemOption) {
             data.each(function (idx) {
                 const rawItem = data.getRawDataItem(idx);
-                if (typeof rawItem === 'object'
-                    && (rawItem as OptionDataItemObject<unknown>).selected
+                if (zrUtil.isObject(rawItem)
+                    && (rawItem as OptionDataItemObject<unknown>)?.selected
```

Comment:
Optional chaining will be transpiled to
```js
rawItem === null || rawItem === void 0 ? void 0 : rawItem.selected;
```
and brings extra codes in the final distribution.

So we limited the use of optional chaining in echarts to keep the distribution as small as possible.

Instead 
```ts
(rawItem && (rawItem as OptionDataItemObject<unknown>).selected)
```
is enough here.

Reply:
get it

Reply:
use zrUtil.isObject(rawItem) replace in here may be more safe?